app/components/questions/RideTripsOhare.js

```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect } from 'react'
+import useFetch from 'use-http'
+import { Table } from '../../components/Table'
+import { Notification } from '../../components/Notification'
+import { FailureNotification } from '../../components/Notification'
+import {
+  Formatter,
+  calculatePercentChange,
+} from '../../site/metrics'
+
+const RIDETRIPS_ENDPOINT = `${process.env.NEXT_PUBLIC_API}/question/ridetrips`;
+  
+  const MAX_PCT_CHANGE = -0.76;
+  const RIDETRIPS_DROPOFF_ALL = [
```

Comment:
Good job setting up the table config! Can you move the indentation of lines 13, 14, and 43 back one indent?